pkg/certificate/manager_test.go

```diff
@@ -318,21 +318,19 @@ func TestIssueCertificate(t *testing.T) {
 
 func TestHandleMRCEvent(t *testing.T) {
```

Comment:
```suggestion
func TestHandleMRCEventAdded(t *testing.T) {
```

I think all of these tests cases cover the type `MCREventAdded`?

Reply:
Good point :)